### PR TITLE
Chore/support scala 2.13

### PR DESCRIPTION
**Pull request by vuhainguyen**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import com.typesafe.sbt.packager.docker.Cmd
 
-val scala212 = "2.12.15"
-val scala213 = "2.13.7"
+val scala212 = "2.12.14"
+val scala213 = "2.13.6"
```
Comment (hakontro) [inline build.sbt:4]: Please add some comments about these version requirements!

File: build.sbt [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 val supportedScalaVersions = List(scala212, scala213)
-val sparkVersion = "3.1.2"
+val sparkVersion = "3.2.0"
```
Comment (hakontro) [inline build.sbt:6]: Yay :sunglasses:

**Review by hakontro — APPROVED** [review]
LGTM! Please also add a version bump and changelog mentioning the new scala 2.13 support etc.

File: build.sbt (hunk 1 above)
Comment (vuhainguyen) [reply build.sbt:4]: I added it directly in your PR https://github.com/cognitedata/cdp-spark-datasource/pull/538/files#r748148476 ;)

**Pull request merged**